### PR TITLE
Create pcl.yml

### PR DESCRIPTION
**Pull request by shawntanzk**
I think I did it right @hkir-dev could you also have a look at this please :) thanks

### CODE REVIEW

File: config/pcl.ym [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- prefix: /tracker/
+  replacement: https://github.com/obophenotype/provisional_cell_ontology/issues
+
+- prefix: /about/
+  replacement: https://github.com/obophenotype/provisional_cell_ontology#readme
+
+## generic fall-through, serve direct from github by default
+- prefix: /
+  replacement: https://raw.githubusercontent.com/insect-morphology/aism/master/
```
Comment (matentzn) [inline config/pcl.ym:40]: typo

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
Please rename the file to the correct extension @shawntanzk then James will take over

**Comment by shawntanzk** [comment]
Thanks @matentzn! sorry about all the mistakes, brains not really on it today apparently >.<

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
No worries this is why there is review :)

**Comment by shawntanzk** [comment]
looks like unresolveable ontobee? I thought stuff gets into ontobee after purls? not sure how to fix >.< We do have it on OLS and can redirect, but there's a bit of a lag in version 
The release number error - ill fix

**Comment by matentzn** [comment]
Haha very fast @shawntanzk. First, this PR must be merged. Then the weekly or so ontobee update triggered. Dont look at ontobee before first week in February.

**Comment by shawntanzk** [comment]
ah was just trying to figure out the error :)

**Pull request merged**